packages/shorebird_cli/lib/src/extensions/arg_results.dart

```diff
@@ -61,6 +61,9 @@ extension CodeSign on ArgResults {
         ? base64Encode(publicKeyFile.readAsBytesSync())
         : null;
   }
+
+  /// Gets a File reference to private from the args if any.
+  File? get privateKeyFile => file('private-key-path');
```

Comment:
It might be better to have these arg names as constants. It would probably make this method unnecessary.

Reply:
Doing this would also make it impossible to mistype the arg name, which I could see happening (we have it hardcoded in a lot of places)

Reply:
Did that! I've also thought in a way to encapsulate the signing logic for use, lmk what you think